app/Http/Controllers/HomeController.php

```diff
@@ -4,13 +4,26 @@
 
 namespace Blumilk\Website\Http\Controllers;
 
+use Blumilk\Website\Models\Reference;
 use Illuminate\Contracts\View\Factory;
 use Illuminate\Contracts\View\View;
+use Illuminate\Support\Facades\App;
 
 class HomeController extends Controller
 {
     public function __invoke(Factory $factory): View
     {
-        return $factory->make("home");
+        $references = Reference::query()->where("published", true)->get();
+        $locale = App::currentLocale();
```

Comment:
```suggestion
        $locale = $request->getLocale();
```
instead of Facade usage